inc/tag_replacer.php

```diff
@@ -210,11 +217,16 @@ public function process_image_tags( $content, $images = [] ) {
 			);
 
 			// If the image is in header or has a class excluded from lazyload, we need to do the regular replace.
-			if ( $images['in_header'][ $index ] ) {
+			if ( $images['in_header'][ $index ] || self::$lazyload_skipped_images < $this->settings->get( 'skip_lazyload_images' ) ) {
```

Comment:
Also, we need to make the limit filtrable for users to be able to adjust this dynamically per context, a filter like `optml_lazyload_images_skip` should be fine.